crates/ironrdp-pdu/src/rdp/suppress_output.rs

```diff
@@ -1,18 +1,22 @@
 use std::io;
 
 use byteorder::{ReadBytesExt as _, WriteBytesExt as _};
-use num_derive::{FromPrimitive, ToPrimitive};
-use num_traits::{FromPrimitive, ToPrimitive};
 
 use crate::{geometry::InclusiveRectangle, PduParsing};
 
 #[repr(u8)]
-#[derive(Debug, PartialEq, Eq, FromPrimitive, ToPrimitive, Clone)]
+#[derive(Debug, PartialEq, Eq, Clone)]
```

Comment:
Note: we are moving away from these traits, `num_derive` and `num_traits` are not removed yet because we still have some legacy code relying on them